### DISCUSSION TITLE
How to use Hooks with Schemathasis docker image

### DISCUSSION THREAD

**Discussion by gnanendra-bogireddy** [schemathesis/schemathesis#2233] [Q&A] [answered]
We are currently using hooks in Schemathesis . It is working fine with CLI usage. Whenever we use the same Docker image it is throwing Module Not Found exception. 

If anyone is using Hooks with Docker can you please share the docker run command.

We are not looking for a docker compose file for this activity. 

What is the default working directory for Schemathesis docker image 

@Stranger6667

**Comment by Stranger6667** [comment, marked as answer]
I thought that the docs covered this use case, but was wrong. I think that the current dir would be /app, so any volume mount should go inside there, then the hooks file should be available from that location. But I am not sure if it would work, I need to experiment with it myself a bit. It would be awesome if anybody could share their experience with this before I do

**Reply by gnanendra-bogireddy** [reply]
I tried to mount Hooks into the app folder and run it.

I am able to run Schemathesis successfully.

Thanks @Stranger6667 for your support.
